### DISCUSSION TITLE
How can I get rid of the index page showing a contact form with public department list?

### DISCUSSION THREAD

**Discussion by CherryDT** [LiveHelperChat/livehelperchat#1854] [Q&A]
I have the problem that when I go to the domain on which I set up LHC, instead of maybe directing me to the admin login (which I could reach by manually using `/index.php/site_admin`), it shows a contact form which, crucially, includes a dropdown of all the departments configured, publicly visible. I would like to get rid of this whole form there, because I don't want somebody extracting the URL from the devtools and visiting the domain to be able to see our list of departments. Is there an option to do that, or should I just redirect this page with `.htaccess`? (Or could that break some other functionality?)

I'm talking about this page on `/`:

![image](https://user-images.githubusercontent.com/6010463/208091768-11acea99-94bd-424a-a46e-27d9e3f188e1.png)

**Comment by remdex** [comment]
In start chat form setting check option that you require department and it will bo gone :)

**Reply by remdex** [reply]
You can also change in settings file https://github.com/LiveHelperChat/livehelperchat/blob/master/lhc_web/settings/settings.ini.default.php#L76-L77

to module `user` and function `login`

**Reply by CherryDT** [reply]
Thank you, I tried that (changing in `settings/settings.ini.php`), but it didn't change and still shows the start chat form. Do I have to do something to apply the change, like clearing some cache somehow?

**Reply by remdex** [reply]
Try to clear cache and also search for siteaccess settings 'eng' each of them can have individual URL.
E.g
https://github.com/LiveHelperChat/livehelperchat/blob/master/lhc_web/settings/settings.ini.default.php#L142-L143
